### ISSUE TITLE
Another python 2 issue

### ISSUE THREAD

**Issue by lukas** [wandb/client#3]
* Weights and Biases version: 0.4.1
* Python version: 2.7
* Operating System: ubuntu

### Description

```
wandb init
```

```Appending to netrc /home/l2k/.netrc
What entity should we scope to? [models]: Traceback (most recent call last):
  File "/home/l2k/.local/bin/wandb", line 11, in <module>
    sys.exit(cli())
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/cli.py", line 38, in wrapper
    return func(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/cli.py", line 298, in init
    result = ctx.invoke(projects, entity=entity)
  File "/home/l2k/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/cli.py", line 38, in wrapper
    return func(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/cli.py", line 86, in projects
    projects = api.list_projects(entity=entity)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/api.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/api.py", line 156, in list_projects
    'entity': entity or self.config('entity')})['models'])
  File "/home/l2k/.local/lib/python2.7/site-packages/gql/client.py", line 52, in execute
    raise Exception(str(result.errors[0]))
Exception: {u'message': u'Resource not found', u'locations': [{u'column': 3, u'line': 2}]}


```

**Comment by vanpelt** [comment]
I need to make these exceptions pretty again, this is actually a 404 from the backend.  Did you type anything, or just use the default 'models' when prompted?

**Comment by vanpelt** [comment]
I fixed the error display in 0.4.2.  Couple things, I'm calling your username "entity" here, but I will change this to username or organization.  I can also grab your username from a more refined login flow.  I'm using "models" as the organization for public models, maybe we should make public models the "zoo" organization?
